_plugins/elixir_school/redirects.rb

```diff
@@ -0,0 +1,23 @@
+module ElixirSchool
+  class Redirects < Jekyll::Generator
+    priority :high
+
+    def generate(site)
+      site.data['redirects'].each do |from, to|
+        site.data['contents'].each do |category, pages|
+          pages.each do |page|
+            if (page == 'home')
```

Comment:
At the moment redirects don't work for translation reports. I guess there should be a special case for them like this one made for index pages.

Reply:
Good catch!  The translation report came after this work had started